src/Type/Accessory/AccessoryArrayListType.php

```diff
@@ -332,6 +333,7 @@ public static function isListTypeEnabled(): bool
 		return self::$enabled;
 	}
 
+	/** @api */
```

Comment:
This doesn't need to be here.

Reply:
I solved it: https://github.com/phpstan/phpstan-src/commit/071320dab361ef35597a524cd472eff3e3531490